auth/src/main/java/com/firebase/ui/auth/viewmodel/AuthViewModelBase.java

```diff
@@ -2,7 +2,6 @@
 
 import android.app.Application;
 
-import com.firebase.ui.auth.AuthUI;
```

Comment:
This change is unrelated to this PR.
I'm not sure how it ended up here but it is harmless, so I guess it's ok to keep it?

Reply:
Yep not a problem.